spec/lib/gakubuchi/template_spec.rb

```diff
@@ -27,74 +48,71 @@
   %w(== === eql?).each do |method_name|
     describe "##{method_name}" do
       subject { template.__send__(method_name, other) }
-      let(:path) { template_root.join('foo.html.erb').to_s }
+      let(:source_path) { 'foo.html.erb' }
 
       context 'other is not an instance of Gakubuchi::Template' do
-        let(:other) { path }
+        let(:other) { source_path }
         it { is_expected.to eq false }
       end
 
       context 'other is an instance of Gakubuchi::Template' do
-        let(:other) { described_class.new(other_path) }
+        let(:other) { described_class.new(other_source_path) }
 
-        context '#pathname is equal to other.pathname' do
-          let(:other_path) { path }
+        context '#source_path is equal to other.source_path' do
+          let(:other_source_path) { source_path }
           it { is_expected.to eq true }
         end
 
-        context '#pathname is not equal to other.pathname' do
-          let(:other_path) { template_root.join('foo/bar.html.erb').to_s }
+        context '#source_path is not equal to other.source_path' do
+          let(:other_source_path) { 'bar/baz.html.erb' }
           it { is_expected.to eq false }
         end
       end
     end
   end
 
-  describe '#destination_pathname' do
-    subject { template.destination_pathname }
-    let(:path) { template_root.join('bar/baz.html.erb').to_s }
+  describe '#digest_path' do
+    subject { template.digest_path }
+
+    context "template does not exist in specified source path" do
```

Comment:
[MUST]
Run rubocop and check warnings.
You'll get `Prefer single-quoted strings when you don't need string interpolation or special symbols.`